indico/modules/categories/serialize.py

```diff
@@ -29,6 +32,7 @@ def serialize_categories_ical(category_ids, user, event_filter=True, event_filte
     :param event_filter_fn: A callable that determines which events to include (after querying)
     :param update_query: A callable that can update the query used to retrieve the events.
                          Must return the updated query object.
+    :param reminder_minutes: If specified, add reminders to all events, handled in ical.py file
```

Comment:
```suggestion
    :param reminder_minutes: If specified, enable reminders in the ical events
```

Better wording, and "handled in ical.py" isn't useful information